src/SerializeResult.cc

```diff
@@ -40,29 +40,43 @@ static sos::Object WrapAnnotation(const snowcrash::SourceAnnotation& annotation)
     return object;
 }
 
+sos::Object drafter::WrapParseResult(const snowcrash::ParseResult<snowcrash::Blueprint>& blueprint,
```

Comment:
I might want to reconsider this name to be `WrapResultRefract`. What do you think @klokane ?